<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GoWindow</class>
 <widget class="QMainWindow" name="GoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>841</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1600</width>
    <height>1600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Go</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>800</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1600</width>
     <height>1600</height>
    </size>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGo">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuQuick_save">
     <property name="title">
      <string>Quick save</string>
     </property>
     <addaction name="onSaveSlot1"/>
     <addaction name="onSaveSlot2"/>
     <addaction name="onSaveSlot3"/>
     <addaction name="onSaveSlot4"/>
     <addaction name="onSaveSlot5"/>
    </widget>
    <widget class="QMenu" name="menuQuick_Load">
     <property name="title">
      <string>Quick Load</string>
     </property>
     <addaction name="onLoadSlot1"/>
     <addaction name="onLoadSlot2"/>
     <addaction name="onLoadSlot3"/>
     <addaction name="onLoadSlot4"/>
     <addaction name="onLoadSlot5"/>
    </widget>
    <addaction name="onLoadGame"/>
    <addaction name="onSaveGame"/>
    <addaction name="menuQuick_save"/>
    <addaction name="menuQuick_Load"/>
   </widget>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <widget class="QMenu" name="menuNew_Game">
     <property name="title">
      <string>New Game</string>
     </property>
     <addaction name="onNew5Game"/>
     <addaction name="onNew9Game"/>
     <addaction name="onNew19Game"/>
     <addaction name="onNewCustomGame"/>
    </widget>
    <widget class="QMenu" name="menuTurn_size">
     <property name="title">
      <string>Turn size</string>
     </property>
     <addaction name="onsSetTurns10"/>
     <addaction name="onSetTurns20"/>
     <addaction name="onSetTurns40"/>
     <addaction name="onSetTurnsCustom"/>
    </widget>
    <addaction name="menuNew_Game"/>
    <addaction name="menuTurn_size"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="onInstructions"/>
   </widget>
   <addaction name="menuGo"/>
   <addaction name="menuGame"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="onLoadGame">
   <property name="text">
    <string>Load game</string>
   </property>
  </action>
  <action name="onSaveGame">
   <property name="text">
    <string>Save game</string>
   </property>
  </action>
  <action name="onNew5Game">
   <property name="text">
    <string>5x5</string>
   </property>
  </action>
  <action name="onNew9Game">
   <property name="text">
    <string>9x9</string>
   </property>
  </action>
  <action name="onNew19Game">
   <property name="text">
    <string>19x19</string>
   </property>
  </action>
  <action name="onsSetTurns10">
   <property name="text">
    <string>10</string>
   </property>
  </action>
  <action name="onSetTurns20">
   <property name="text">
    <string>20</string>
   </property>
  </action>
  <action name="onSetTurns40">
   <property name="text">
    <string>40</string>
   </property>
  </action>
  <action name="onSetTurnsCustom">
   <property name="text">
    <string>custum</string>
   </property>
  </action>
  <action name="onNewCustomGame">
   <property name="text">
    <string>custom</string>
   </property>
  </action>
  <action name="onInstructions">
   <property name="text">
    <string>Instructions</string>
   </property>
  </action>
  <action name="onSaveSlot1">
   <property name="text">
    <string>Slot 1</string>
   </property>
  </action>
  <action name="onSaveSlot2">
   <property name="text">
    <string>Slot 2</string>
   </property>
  </action>
  <action name="onSaveSlot3">
   <property name="text">
    <string>Slot 3</string>
   </property>
  </action>
  <action name="onSaveSlot4">
   <property name="text">
    <string>Slot 4</string>
   </property>
  </action>
  <action name="onSaveSlot5">
   <property name="text">
    <string>Slot 5</string>
   </property>
  </action>
  <action name="onLoadSlot1">
   <property name="text">
    <string>Slot 1</string>
   </property>
  </action>
  <action name="onLoadSlot2">
   <property name="text">
    <string>Slot 2</string>
   </property>
  </action>
  <action name="onLoadSlot3">
   <property name="text">
    <string>Slot 3</string>
   </property>
  </action>
  <action name="onLoadSlot4">
   <property name="text">
    <string>Slot 4</string>
   </property>
  </action>
  <action name="onLoadSlot5">
   <property name="text">
    <string>Slot 5</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
